core/sentry_bot.py

```
@@ -126,6 +175,105 @@ async def handle_log_analysis(message, say, context):
                 logger.error(f"Log analysis failed: {e}")
                 await say(f"❌ Log analysis failed: {str(e)}")
                 
+        @self.app.event("app_mention")
+        async def handle_mention(body, say, context, client: WebClient, logger):
+            """Handle direct mentions of the bot for research or thread summarization."""
+            event = body['event']
+            text = event['text']
+            user_id = event['user']
```

Ruff (0.12.2)

Local variable `user_id` is assigned to but never used

Remove assignment to unused variable `user_id`

(F841)